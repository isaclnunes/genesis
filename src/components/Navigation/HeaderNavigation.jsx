import React      from 'react';
import { Link }   from 'react-router';
import { Navbar } from 'react-bootstrap';

/* eslint semi: 0 */
const HeaderNavigation = () =>
  <Navbar fixedTop>
    <Navbar.Header>
      <Navbar.Brand>
        <Link to="/">Tiago Pina</Link>
      </Navbar.Brand>
      <Navbar.Toggle />
    </Navbar.Header>
  </Navbar>

export default HeaderNavigation;
