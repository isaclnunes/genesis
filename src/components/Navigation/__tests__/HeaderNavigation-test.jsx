import React from 'react';
import ReactDom from 'react-dom';
import renderer from 'react-test-renderer';
import jest from 'jest';

import HeaderNavigation from '../HeaderNavigation';

describe('HeaderNavigation', () => {

  test('HeaderNavigation renders correctly', () => {
    const component = renderer.create(
      <HeaderNavigation />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
