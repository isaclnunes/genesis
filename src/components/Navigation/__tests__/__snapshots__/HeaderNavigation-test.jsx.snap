exports[`HeaderNavigation HeaderNavigation renders correctly 1`] = `
<nav
  className="navbar navbar-default navbar-fixed-top">
  <div
    className="container">
    <div
      className="navbar-header">
      <a
        className="navbar-brand"
        onClick={[Function]}
        style={Object {}}>
        ReactWeather
      </a>
      <button
        className="navbar-toggle collapsed"
        onClick={[Function]}
        type="button">
        <span
          className="sr-only">
          Toggle navigation
        </span>
        <span
          className="icon-bar" />
        <span
          className="icon-bar" />
        <span
          className="icon-bar" />
      </button>
    </div>
  </div>
</nav>
`;
